.github/workflows/ci.yaml

```diff
@@ -180,26 +199,19 @@ jobs:
         neuro secret add azure_sas_token $AZURE_SAS_TOKEN
         neuro --color=no config show
 
-    - name: Configure GCP access
-      uses: google-github-actions/setup-gcloud@master
-      with:
-        project_id: ${{ secrets.E2E_COOKIECUTTER_GCP_PROJECT_ID }}
-        service_account_key: ${{ secrets.E2E_COOKIECUTTER_GCP_SA_KEY }}
-        export_default_credentials: true
-
-    - name: Configure AWS access
-      uses: aws-actions/configure-aws-credentials@v1
-      with:
-        aws-access-key-id: ${{ secrets.E2E_COOKIECUTTER_AWS_KEY_ID }}
-        aws-secret-access-key: ${{ secrets.E2E_COOKIECUTTER_AWS_ACCESS_KEY }}
-        aws-region: us-east-1
-
     - name: Run tests
       env:
         COLOR: 'yes'
-        DOCKER_CI_USERNAME: ${{ secrets.DOCKER_CI_USERNAME }}
-        DOCKER_CI_TOKEN: ${{ secrets.DOCKER_CI_TOKEN }}
+        DOCKER_CI_USERNAME: ${{ secrets.DOCKERHUB_CI_USERNAME }}
+        DOCKER_CI_TOKEN: ${{ secrets.DOCKERHUB_CI_TOKEN }}
+      shell: bash
       run: |
+        # Work-around, since gsutil does not support Python 3.10 yet.
```

Comment:
I wonder what users with Python 3.10 are supposed to do when using `gsutil` with neuro. I mean if they take neuro-extras image and try to use it, they have to know the workaround you mentioned here.
Also, if they follow our tutorial on GCP on the website and use Python 3.10 (which we claim to support), they will get into the same issue.

So, perhaps we should:

1. Not support 3.10 in `neuro-extras` yet as it might confuse the users. Or that may be okay, since we have no users :) I'm leaving this decision up to you.
2. Update the docs re: GCP on the website.

Reply:
Python version in neuro-extras image is [unchanged](https://github.com/neuro-inc/neuro-extras/blob/ys/fix-tests/Dockerfile#L1).
This is the work-around for the following case:
- user has Python 3.10 on his host machine
- he installed NE into his env
- wants to do `ne data cp gs:// /some/local/path`

Re docs, you meant [this](https://docs.neu.ro/toolbox/accessing-remote-storage/accessing-object-storage-in-gcp#creating-a-bucket-and-granting-access) topic?

Reply:
1. How does the workaround help the user? I understand that helps to run the CI, but we should let users know they should do the same.
2. Yes, that topic in the docs

Reply:
1. It actually will not help, until we find a way to install all external libs together with the NE. For this iteration, I would suggest pointing it out in the project's `Readme.md`, WDYT?
2. This topic is outdated, unfortunately. AFAIK, it will be replaced with the description of our own buckets usage.

Reply:
1. Sounds good to me
2. If it's outdated, it should be removed, right?

Reply:
1. done in e672b56